### PR TITLE
Add NULL values to count of HC encounters

### PR DESCRIPTION
**Pull request by adamhstewart**
Prior to May 2021 health center encounters are NULL

### CODE REVIEW

File: server/hedley/modules/custom/hedley_admin/scripts/generate-anc-report.php [hunk 1]
```diff
@@ -54,8 +54,8 @@
        field_data_field_prenatal_encounter_type fdfpet ON p.entity_id = fdfpet.entity_id
    WHERE
      p.bundle = 'prenatal_encounter' AND
-     fdfpet.field_prenatal_encounter_type_value='nurse' AND
-     date(edd.field_expected_date_concluded_value) > DATE_ADD(CURDATE(), INTERVAL 30 DAY)
+     (fdfpet.field_prenatal_encounter_type_value='nurse' OR fdfpet.field_prenatal_encounter_type_value is NULL) AND
+     date(edd.field_expected_date_concluded_value) > DATE_ADD('2021-05-30', INTERVAL 30 DAY)
```
Comment (adamhstewart) [inline server/hedley/modules/custom/hedley_admin/scripts/generate-anc-report.php]: needs to be reverted.

**Comment by adamhstewart** [comment]
https://github.com/Gizra/ihangane/issues/2946

**Comment by adamhstewart** [comment]
@AronNovak could you check the work here, but also give a hand with the code formatting - I ran phpcbf and it didn't help

What we did:
- Counted NULL values for the ANC report
- Provided a limit date as an argument from the robo command (default to today)
- Also provide a limit date for the pregnancy report.

**Comment by AronNovak** [comment]
@adamhstewart I restructured the code to be more clean and it has the side-effect of passing phpcs.
I started to implement `ddev phpcs`, but I don't want to waste more time on it, if you think it worth +1h, I will continue the effort.

**Comment by adamhstewart** [comment]
> if you think it worth +1h, I will continue the effort.

Thanks @AronNovak - let's do that.

**Comment by adamhstewart** [comment]
@AronNovak this restructured code removes the interval limits  so that all the queries yield the same results. See for instance:

image

**Comment by adamhstewart** [comment]
@AronNovak added back the proper limits for active and completed preganancies.

**Comment by AronNovak** [comment]
@adamhstewart 
I know it's not the most serious issue, but I started with an easy task, local `ddev phpcs` works like on other projects:
image

**Comment by anvmn** [comment]
Replaced by #197. Closing...

**Pull request closed without merging**